priv/template/lib/application_name/endpoint.ex

```diff
@@ -4,7 +4,8 @@ defmodule <%= application_module %>.Endpoint do
   # Serve at "/" the given assets from "priv/static" directory
   plug Plug.Static,
     at: "/", from: :<%= application_name %>,
-    only: ~w(css images js favicon.ico robots.txt)
+    only: ~w(css images js favicon.ico robots.txt
```

Comment:
I though we agreed to compile everything into `css` and `js` so we don't need the extra line below?